.github/workflows/mono-uninstall.sh

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
```

Comment:
This file is not part of linux build, could you remove it from this PR?  Probably you need to save it for later.

Reply:
It's a part of MacOS build.